app/grants/templates/grants/components/form_wrapper.html

```diff
@@ -7,6 +7,7 @@ <h4>[[heading]]</h4>
       </template>
       <slot></slot>
     </div>
+    <slot name="footer"></slot>
```

Comment:
what's this for ?

Reply:
This is for additional content that we display below the wizard, if it is the case.
We have a use case where we display fee information for bounty payments in the payment step only.